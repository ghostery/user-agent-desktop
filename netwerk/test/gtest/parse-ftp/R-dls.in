README              763  Information about this server
bin/                  -  
etc/                  =  
ls-lR                 0  
ls-lR.Z               3  
pub/                  =  Public area
usr/                  -  
morgan               14  -> ../real/morgan
TIMIT.mostlikely.Z
                  79215  
README              763  Feb  3 18:21  Information about this server
bin/                  -  Apr 28  1994
etc/                  =  11 Jul 21:04
ls-lR                 0   6 Aug 17:14
ls-lR.Z               3  05 Sep 1994
pub/                  =  Jul 11 21:04  Public area
usr/                  -  Sep  7 09:39
morgan               14  Apr 18  9:39  -> ../real/morgan
TIMIT.mostlikely.Z
                  79215  Jan  4  6:45
                      
*** CAUTION while editing this sample LISTing - 
*** Lines contain trailing whitespace and/or '\r's
