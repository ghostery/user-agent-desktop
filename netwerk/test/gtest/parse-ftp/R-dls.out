<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

00-00-0000  00:00:00                  763 README
00-00-0000  00:00:00       <DIR>          bin
00-00-0000  00:00:00       <DIR>          etc
00-00-0000  00:00:00                    0 ls-lR
00-00-0000  00:00:00                    3 ls-lR.Z
00-00-0000  00:00:00       <DIR>          pub
00-00-0000  00:00:00       <DIR>          usr
00-00-0000  00:00:00       <JUNCTION>     morgan -> ../real/morgan
00-00-0000  00:00:00                79215 TIMIT.mostlikely.Z
02-03-2002  18:21:00                  763 README
04-28-1994  00:00:00       <DIR>          bin
07-11-2002  21:04:00       <DIR>          etc
08-06-2001  17:14:00                    0 ls-lR
09-05-1994  00:00:00                    3 ls-lR.Z
07-11-2002  21:04:00       <DIR>          pub
09-07-2001  09:39:00       <DIR>          usr
04-18-2002  09:39:00       <JUNCTION>     morgan -> ../real/morgan
01-04-2002  06:45:00                79215 TIMIT.mostlikely.Z
